presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4254,6 +4254,19 @@ private void schemaMismatchesWithDereferenceProjections(HiveStorageFormat format
         finally {
             assertUpdate("DROP TABLE IF EXISTS evolve_test");
         }
+
+        // Add extra subfield
```

Comment:
nit: something like `//  Verify field access when the row evolves without changes to field type` would be more descriptive.